ideas/120-swarm-process.md

```diff
@@ -0,0 +1,146 @@
+## Preamble
+
+    Idea: 120-swarm-process
+    Title: Formalize swarm process
+    Status: In Progress
+    Created: 2018-03-30
+    Requires (*optional):
+    Replaces (*optional):
+
+## Summary
+Swarms are the primary structure we use to turn ideas into reality at Status.
+People who are excited about bringing an idea to fruition commit to it and make
+sure it happens.
+
+At an individual level, this ensures individual contributors are enabled to
+achieve autonomy, mastery and purpose, while still ensuring accountability.
+
+At a product team / organizational level, swarms allow us to see what is going
+on and which OKRs are targetted, which are not, and generally track progress at
+a reasonable level of granularity.
+
+However, the process for creating, running and completing swarms is currently
+ad-hoc, lacking in accountability and follow-up, and knowledge of how to do this
+properly isn't evenly spread throughout the organization. This swarm will fix
+this.
+
+## Swarm Participants
+- Lead Contributor: @oskarth
+- Evaluation ('knowledge spreading'): Voting mechanism (Polly survey)
+- Evaluation ('OKR coverage'): OKR spreadsheet coverage and/or @chad and/or @nabil (TBC)
+- Evaluation ('enable experimentation'): @martin
+- Contributor: @martin
+
+- Possibly UX for communicating things during Town Hall, etc.
+
+## Product Overview
+There are two constraints that govern how this structure is designed and
+cultivated. One is that Status the company wants to ship a product, and the
+other is creating a decentralized autonomous organization. Since we are moving
+towards the latter, the structures we are building must be compatible with this
+latter vision, while still being mindful of the first.
+
+The primary difference between the two is that in the former, we are speaking on
+behalf of users, and people are generally compensated on a salary or salary-like
+basis.
+
+We thus distinguish between swarms (Type-1) as a general concept for bringing
+ideas to fruition and getting compensated by it, and 'Status LLC using swarms'
+(Type-2). The fatter might have more constraints given its immediate goals
```

Comment:
@oskarth, did you mean "later" or "fatter" is a correct word? :)

Reply:
Typo indeed!

Reply:
fixed https://github.com/status-im/ideas/blob/master/ideas/120-swarm-process.md